deployment/src/main/java/io/quarkiverse/renarde/deployment/RenardeProcessor.java

```diff
@@ -216,8 +216,11 @@ void setupSecurity(LaunchModeBuildItem launchMode, Capabilities capabilities,
             // those are the better defaults
             defineUnlessPresent("mp.jwt.token.header", "Cookie", config, runtimeConfigurationBuildItem);
             defineUnlessPresent("mp.jwt.token.cookie", "QuarkusUser", config, runtimeConfigurationBuildItem);
+            defineUnlessPresent("renarde.redirect-location", "quarkus-redirect-location", config,
+                    runtimeConfigurationBuildItem);
         }
         // Apparently, no OIDC capability to check
+        defineUnlessPresent("quarkus.oidc.authentication.cookie-suffix", "q_session", config, runtimeConfigurationBuildItem);
```

Comment:
Why is this needed? This is the default, IIUC, so we should not need to override it.